lib/zlib.js

```diff
@@ -95,7 +95,7 @@ const {
 
 // Translation table for return codes.
 const codes = {
-  Z_OK: constants.Z_OK,
+  __proto__: null, Z_OK: constants.Z_OK,
```

Comment:
You'll want to make sure you add a new line in cases like this. (I'm surprised some other lint rule doesn't complain TBH.)

Reply:
(Oh, I see ljharb has already commented on a few of these.)

Reply:
fixed the rule to add multiline when needed